tests/test_imports.py

```diff
@@ -0,0 +1,43 @@
+import platform
+import sys
+
+import pytest
+
+
+def test___all__(pytester: pytest.Pytester) -> None:
+    """See https://github.com/aio-libs/aiohttp/issues/6197"""
+    pytester.makepyfile(
+        test_a="""
+            from aiohttp import *
+            assert 'GunicornWebWorker' in globals()
+        """
+    )
+    result = pytester.runpytest("-vv")
+    result.assert_outcomes(passed=0, errors=0)
+
+
+def test_web___all__(pytester: pytest.Pytester) -> None:
+    pytester.makepyfile(
+        test_b="""
+            from aiohttp.web import *
```

Comment:
Does this test anything https://github.com/aio-libs/aiohttp/blob/77d3d68/tests/test_circular_imports.py doesn't? Maybe just parametrize that test to check both `import ...` and `from ... import ...` syntaxes?